### PR TITLE
Add a configuration option to disable XHR wrapping

### PR DESCRIPTION
**Pull request by danielmohacsi**
The new Sentry client offers a bunch of interesting features, eg:
- console recording
- stack trace recording
- POST support
And a lot more!

The bad thing is that this client also wraps XHR calls, and we have had a few issues with customers complaining about us hijacking their AJAX calls.
This little PR adds a simple configuration property, `wrapXhr`, which lets us turn this automatic wrapping off.

[They promised](https://github.com/getsentry/raven-js/issues/627) to make this automatic wrapping configurable, but in the meantime, it would be nice to be able to use the new features.

We can go back to their official release, once they let us to opt-out from XHR wrapping.

### CODE REVIEW

File: dist/raven.js [hunk 1]
```diff
@@ -1284,10 +1283,14 @@ Raven.prototype = {
...
+        var exception = data.exception && data.exception.values[0];
         this.captureBreadcrumb({
             category: 'sentry',
-            message: data.message,
-            event_id: data.event_id
+            message: exception
+                ? (exception.type ? exception.type + ': ' : '') + exception.message
+                : data.message,
+            event_id: data.event_id,
+            level: data.level || 'error' // presume error unless specified
```
Comment (rgabo) [inline dist/raven.js]: Does the change have anything to do with *"Add a configuration option to disable XHR wrapping"*?

Comment (danielmohacsi) [inline dist/raven.js]: This file was added after the result of the `grunt dist` command. I did not add these changes to the code, it seems that they had a few features on `master` that have not yet been released.

**Comment by rgabo** [comment]
LGTM, won't merge though as you might want to use from branch until upstream implements/merges.

**Comment by rgabo** [comment]
Does that mean it was never merged upstream?

**Comment by danielmohacsi** [comment]
It won't be. https://github.com/getsentry/raven-js/issues/627
<https://github.com/getsentry/raven-js/issues/627>
<https://github.com/getsentry/raven-js/issues/627>
<https://github.com/getsentry/raven-js/issues/627>
And I'm working on a different feature that enables/disables wrapping
globally, not one-by-one. Bebe raised that we're overriding builtin
features such as console logging, and set timeout.

**Pull request merged**